content/docs/dvclive/dvclive-with-dvc.md

```diff
@@ -3,17 +3,29 @@
 Even though Dvclive does not require DVC, they can integrate in several useful
 ways.
 
-> In this section we will modify the [basic usage example](/doc/dvclive/usage)
-> to see how DVC can cooperate with Dvclive module.
+> In this section we reuse the finished
+> [basic usage example](/doc/dvclive/usage) to see how DVC can cooperate with
+> Dvclive.
 
 ```python
 # train.py
 
+import dvclive
+from keras.callbacks import Callback
 from keras.datasets import mnist
 from keras.models import Sequential
 from keras.layers.core import Dense, Activation
 from keras.utils import np_utils
 
+
+class MetricsCallback(Callback):
+    def on_epoch_end(self, epoch: int, logs: dict = None):
+        logs = logs or {}
+        for metric, value in logs.items():
+            dvclive.log(metric, value)
+        dvclive.next_step()
+
+
 def load_data():
     (x_train, y_train), (x_test, y_test) = mnist.load_data()
```

Comment:
TBH I'd even considering replacing both functions and some imports with `...`, or at least their contents (as if collapsed in an IDE) — in the latter case maybe also the class.

Reply:
I have been including all code, because personally I prefer tutorials that way, so my view on that is skewed. If we collapse imports and methods in other docs, sure.

Reply:
OK but it's not really a tutorial, I think. I.e. the specific It's a guide, and it's repeated from https://dvc.org/doc/dvclive/usage which presumably most people will read first.

BTW I'd argue to collapse or at least hide some details in both guides 😬 — not in this PR. And yes we def. shorten code blocks in most docs except in Get Started mainly, which is a step-by-step walkthrough we expect users to actually follow closely.